pkg/serverless/executioncontext/executioncontext.go

```diff
@@ -25,6 +25,8 @@ type ExecutionContext struct {
 	coldstart          bool
 	startTime          time.Time
 	endTime            time.Time
+
+	persistedStateFilePath string
```

Comment:
can't this be assigned with default `default:""/tmp/dd-lambda-extension-cache.json""` ?

Reply:
Oh wow yes, I didn't know that feature even existed! Thanks for pointing it out!